assignments/sidharth/week1/1.8.txt

```diff
@@ -0,0 +1,12 @@
+(define (cubert-iter guess x)
+ (if (good-enough? guess x)
+  guess
+   (cubert-iter (improve guess x)
+    x)))
+(define (improve guess x)
+ (/ (+ (/ x (* guess guess)) (* 2 y)) 3)
```

Comment:
I think you meant "guess" instead of "y". And this statement is missing a closing paren ).